docs/storage.md

```diff
@@ -35,35 +35,62 @@ server.run(8000);
 
 ### Firebase
 
-First, install the necessary packages:
+First, install the Firebase Admin Node.js SDK:
 
 ```
-npm install --save firebase
+npm install --save firebase-admin
```

Comment:
Just to double-check, users don't have to also install `firebase` (i.e. `npm install --save firebase firebase-admin`)?

Reply:
In practical terms, no: I expect most users will install `firebase-admin`, and set `adminClient` to `true` for their one `Firebase` instantiation, which will not require installing `firebase`.  If they set `adminClient` as `false`, they'll need to install `firebase` (but again not both).  If they create two Firebase objects with both `adminClient` states, they'll probably figure out they need to install both. ;)

Reply:
OK, thanks for the clarification!